src/lesson13/validator.ts

```diff
@@ -0,0 +1,45 @@
+export type Value = string | number | boolean;
+
+export class Validator {
+  errors: { [key: string]: Error } = {};
+  values: { [key: string]: Value } = {};
+
+  updateField(key: string, value: Value) {
+    this.values[key] = value;
+  }
+
+  hasErrors() {
+    let isValid = true;
```

Comment:
for (key in this.errors) {
      if (this.errors[key]) {
        return false;
      }
}

Reply:
done